usb.c

```diff
@@ -953,10 +953,10 @@ static int usb_fill_context_info(struct iio_context_info *info,
 		unsigned int interface)
 {
 	struct libusb_device_descriptor desc;
-	char manufacturer[64], product[64];
+	char manufacturer[64], product[64], serial[64];
 	char uri[sizeof("usb:127.255.255")];
 	char description[sizeof(manufacturer) + sizeof(product) +
-		sizeof("0000:0000 ( )")];
+		sizeof(serial) + sizeof("0000:0000 ( )")];
```

Comment:
it should be sizeof("0000:0000 ( ), serial=")
Otherwise the buffer is too small.